File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex3.java
//Importando Scanner
import java.util.Scanner;

public class Ex3 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Troca de 8 valores de um vetor\n\n");
		
		//vars
		int v[] = new int[16];
		int i, aux;
		
		//Entrada
		for( i=0; i<16; i++)
		{
			System.out.print("Insira v[" + i + "]: ");
			v[i] = ent.nextInt();
		}
		
		//Processamento
		for( i=0; i<8; i++)
		{
			aux = v[i];
			v[i] = v[i+8];
			v[i+8] = aux;
		}
		
		//Saída
		System.out.println("Vetor com as 8 posições iniciais trocadas pelas finais");
		for( i=0; i<16; i++)
		{
			System.out.printf("%d  ", v[i]);
		}
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex3.c
#include <stdio.h>

main()
{
    printf("Troca dos 8 elementos finais pelos iniciais de um vetor\n\n");

    //vars
    int v[16], i, aux;

    //Entrada
    for( i=0; i<16; i++)
    {
        printf("Digite v[%i]: ", i);
        scanf("%d", &v[i]);
    }

    //Processamento
    for(i=0; i<8; i++)
    {
        aux = v[i];
        v[i] = v[i+8];
        v[i+8] = aux;
    }

    //Saída
    printf("Vetor Resultante: ");
    for( i=0; i<16; i++)
    {
        printf("%d  ", v[i]);
    }

}
